### DISCUSSION TITLE
Pydantic Settings doc content fix

### DISCUSSION THREAD

**Discussion by johndutchover** [pydantic/pydantic#7950] [Question] [answered]
**Question**
Can content **_from_** [pydantic/pydantic-settings - index.md](https://raw.githubusercontent.com/pydantic/pydantic-settings/main/docs/index.md) **index.html** be appended to `pydantic_settings.md` in order for me to resolve minor markdown issues in docs?

**Findings**
I found some doc content issues on the [Settings Management - Pydantic](https://docs.pydantic.dev/latest/concepts/pydantic_settings/) page. However, the [pydantic_settings.md](https://github.com/pydantic/pydantic/blob/9c91f856d8710a4f5e8cf34a7662b9324020db1e/docs/concepts/pydantic_settings.md) currently renders:
> <img width="785" alt="pydantic-external-markdown" src="https://github.com/pydantic/pydantic/assets/1088664/d65de90d-7c1e-41f0-b0dd-1afde7c85884">

**Comment by hramezani** [comment, marked as answer]
Why don't you submit a PR to fix the issues in [pydantic-settings](https://github.com/pydantic/pydantic-settings)

**Reply by johndutchover** [reply]
Will do, I wanted to be certain about the flow. Thanks.

**Reply by johndutchover** [reply]
https://github.com/pydantic/pydantic-settings/pull/182
